packages/aragon-wrapper/src/core/proxy/index.js

```diff
@@ -47,7 +47,17 @@ export default class Proxy {
       throw new Error(`No method named ${method} on ${this.address}`)
     }
 
-    return this.contract.methods[method](...params).call()
+    const { from } = params[params.length - 1]
+
+    if (typeof lastParam === 'object' && lastParam !== null) {
+      // Add `from` to call params only if it's defined, can be easily extended for more params
+      return this.contract.methods[method](...params.slice(0,-1)).call({ 
+        ...(from && { from }) 
+      })
+    }
```

Comment:
A few nitpicks on the above block:

- `lastParam` doesn't seem to be defined
- `...params.slice(0,-1)` doesn't need the spread, since it makes a new array already
	- I'm not sure why it was being spread in the original case, and we can probably remove that
- We can probably just pass in the entire `lastParam` to `call()` so any of the [optional parameters](http://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#id16) are supported
- Would be great to document here: https://github.com/aragon/aragon.js/blob/master/docs/APP.md#call